### PR TITLE
[3211] Add new anomaly

### PR DESCRIPTION
**Pull request by emmiegit**
Screenshot of the iframe contents, no window formatting:

image

Image source: https://commons.wikimedia.org/wiki/File:Bengal_Cat_Babou_Front_Face_Almost_3_Years_Old.jpg (CC-BY-SA 4.0)

### CODE REVIEW

File: articles/scp-3211/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SCP-3211
 
-This is the sub-repository for [SCP-3211](https://www.scpwiki.com/scp-3211),
+This is the sub-repository for [SCP-3211](http://www.scpwiki.com/scp-3211),
```
Comment (rossjrw) [inline articles/scp-3211/README.md:3]: I'd prefer that PRs be kept on topic (there are also other places where URLs of this form would need to be changed for consistency)

File: articles/scp-3211/src/config.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export const anomalyNames = <const>[
   "base", // Base anomaly must come first
+  "cat",
```
Comment (rossjrw) [inline articles/scp-3211/src/config.ts:3]: Anomalies should be declared in the order they're added - this list determines what order the buttons appear in on the selection menu, and I want older ones to appear first

File: articles/scp-3211/src/en/anomalies/cat.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Anomaly } from "../../anomaly"
+
+export default new Anomaly(false, {
+  imageUrl: "cat.jpg",
+  imageCaption: "SCP-3211 photographed in containment.",
+  objectClass: "Safe",
+  physicalDescription: `
+    SCP-3211 is a male Bengal cat that is estimated to be about
+    3 years old. It appears with a color and metal tag around its neck.
```
Comment (rossjrw) [inline articles/scp-3211/src/en/anomalies/cat.ts:9]: "color and metal"? Possibly 'collar'?

What does "it appears" mean here?

File: articles/scp-3211/src/en/anomalies/cat.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    Unless being continually observed, the cat may teleport to another
+    position within its containment chamber. The animal does not apparently
+    notice the displacement, continuing whatever activity it was when observed.
```
Comment (rossjrw) [inline articles/scp-3211/src/en/anomalies/cat.ts:13]:
```suggestion
    notice the displacement, continuing whatever activity it was doing when observed.
```

File: articles/scp-3211/src/en/anomalies/cat.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+export default new Anomaly(false, {
+  imageUrl: "cat.jpg",
+  imageCaption: "SCP-3211 photographed in containment.",
+  objectClass: "Safe",
+  physicalDescription: `
+    SCP-3211 is a male Bengal cat that is estimated to be about
+    3 years old. It appears with a color and metal tag around its neck.
+
+    Unless being continually observed, the cat may teleport to another
+    position within its containment chamber. The animal does not apparently
```
Comment (rossjrw) [inline articles/scp-3211/src/en/anomalies/cat.ts:12]:
```suggestion
    position within its containment chamber. It does not apparently
```

Is this a universal truth - does it never notice the change? Current wording seems to refer to a single event

File: articles/scp-3211/src/en/anomalies/cat.ts [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  informalDescription: "a cat",
+  formalDescription: "a domestic cat",
+  writtenObservation: `
+    It's a cat. Cute and fluffy, but it jumps around sometimes when I blink
+    or look away. Kinda weird.
+  `,
+  data: {
+    spectrophotometry: "SCP-3211 displayed absorbances as expected",
+    mass: "4.7 kg",
+    hume: "SCP-3211 has a Hume reading slightly higher with baseline reality",
```
Comment (rossjrw) [inline articles/scp-3211/src/en/anomalies/cat.ts:24]:
```suggestion
    hume: "SCP-3211 has a Hume reading slightly higher than baseline reality",
```

**Review by rossjrw — COMMENTED** [review]
Regarding the image, because GitHub lacks a way to comment directly on the file: it should be 300x240px for consistency with the other images, or at least the same ratio (I appreciate that because the other images are WEBP they're hard to preview and compare against - I'll probably switch back to PNG/JPG at some point, I don't think I'm gaining much for it).

Good to know that the setup makes it easy to add stuff!

So my problem with this specific sub-anomaly is that it's not particularly unique/interesting. We already have a lot of observational+teleportation anomalies, including a bunch even within S1 ([810](http://scp-wiki.wikidot.com/scp-810), [650](http://www.scpwiki.com/scp-650), [689](http://www.scpwiki.com/scp-689)), even sort of including [our lord and saviour](http://www.scpwiki.com/scp-173). That's not to say that any of the existing sub-anomalies are particularly unique, though if that's the case, I'm loathe to add even more on the pile - I'd first want to trim out the poorer ones (like the pigeon) before I add any more in preparation for a rewrite.

I guess all that is to say that I'm not really looking for new sub-anomalies right now - but thanks for the thought!

File: articles/scp-3211/README.md (hunk 1 above)
Comment (emmiegit) [reply articles/scp-3211/README.md:3]: Sorry, yeah, noticed it and added it in. You should still add make this change due to the TLS issue.

File: articles/scp-3211/src/config.ts (hunk 2 above)
Comment (emmiegit) [reply articles/scp-3211/src/config.ts:3]: Ah I see, apologies. Maybe leave a comment saying "add at the end"?

File: articles/scp-3211/src/en/anomalies/cat.ts (hunk 3 above)
Comment (emmiegit) [reply articles/scp-3211/src/en/anomalies/cat.ts:9]: Wow, I'm tired and misspelled "collar". sad

**Pull request closed without merging**